pokemonCards/pokemoncard_api/views.py

```diff
@@ -48,4 +65,33 @@ def get_serializer_class(self):
             return PokemonCollectionWriteSerializer
         else: 
             return PokemonCollectionReadSerializer
-            
\ No newline at end of file
+
+class UserViewset(viewsets.ModelViewSet):
+    serializer_class = UserSerializer
+
+    def get_queryset(self):
+        queryset = User.objects.all()
+        return queryset 
+
+class LogInViewset(viewsets.ModelViewSet):
+    serializer_class = LogInSerializer
+
+    def get_queryset(self):
+        queryset = User.objects.all()
+        return queryset
+
+    def update(self, request, *args, **kwargs):
+        
+        password = request.data.get("password")
+        try:
+            userInDB = User.objects.get(password=password)
+        except:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+        if userInDB:
```

Comment:
we should probably add a return Response(status=status.HTTP_404_NOT_FOUND) in case this if statement isn't true